grails-app/controllers/org/broadinstitute/orsp/api/ProjectController.groovy

```diff
@@ -80,15 +79,23 @@ class ProjectController extends AuthenticatedController {
         }
     }
 
-    @Override
-    handleIntake(String key) {
+
+    def update() {
+        Gson gson = new Gson()
+        Map project = gson.fromJson(gson.toJson(request.JSON), Map.class)
```

Comment:
Nit: I've seen this same code in other places. Can we abstract it out to a utility?

Reply:
Also - I'm very suspicious of untyped collections. I strongly prefer that we try to specify types, even if it's `Map<String, Object>` and let `IssueService.updateIssue` handle the particulars of determining what the `Object` part of that is.